ui/src/app/utils/access-utils.tsx

```diff
@@ -461,10 +461,19 @@ const withInvalidDateHandling = (date) => {
 export const hasExpired = (expiration: number): boolean =>
   !!expiration && getWholeDaysFromNow(expiration) < 0;
 
-export const isExpiringOrExpired = (expiration: number): boolean =>
-  !!expiration &&
-  getWholeDaysFromNow(expiration) <=
-    serverConfigStore.get().config.accessRenewalLookback;
+export const isExpiringOrExpired = (
+  expiration: number,
+  module: AccessModule
+): boolean => {
+  const moodleModules = [
```

Comment:
nit: Moodle is not incorrect, but maybe `trainingModules`? Since we don't otherwise mention moodle here.